test/test_io.jl

```diff
@@ -69,7 +69,7 @@ end
         detno = Int32[1],
         thit = VectorOfVectors([T[0] * u"s"]),
         edep = VectorOfVectors([T[1] * u"eV"]),
-        pos = VectorOfVectors([[SVector{3, T}.(0.01, 0.01, 0.01) * u"m"]])
```

Comment:
Why? The method with `SVectors` and units should still work.
Then, we just need to call `to_internal_units` or `_parse_value` somewhere, where we're calling `ustrip` now.